src/runtime/stores/locale.ts

```diff
@@ -32,18 +32,21 @@ export function getCurrentLocale() {
   return current;
 }
 
-$locale.subscribe((newLocale: string) => {
+internalLocale.subscribe((newLocale: string | null | undefined) => {
   current = newLocale;
 
-  if (typeof window !== 'undefined' && newLocale !== null) {
+  if (typeof window !== 'undefined' && newLocale != null) {
```

Comment:
This line is extra change to runtime behavior. It affects when `newLocale` is `undefined`.